File: samples/petstore/scala/src/main/scala/com/wordnik/petstore/model/Tag.scala
package com.wordnik.petstore.model

import scala.reflect.BeanProperty

class Tag {
  @BeanProperty var id: Long = _
  @BeanProperty var name: String = _
  override def toString: String = {
    val sb = new StringBuilder
    sb.append("class Tag {\n")
    sb.append("  id: ").append(id).append("\n")
    sb.append("  name: ").append(name).append("\n")
    sb.append("}\n")
    sb.toString
  }
}

File: samples/client/petstore/csharp/src/Com/Wordnik/Petstore/Model/Tag.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Wordnik.Petstore.Model {
  public class Tag {
    public long id { get; set; }

    public string name { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class Tag {\n");
      sb.Append("  id: ").Append(id).Append("\n");
      sb.Append("  name: ").Append(name).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }
